test/controllers/api/v2/repositories_controller_test.rb

```diff
@@ -468,6 +469,7 @@ def test_show_sync_plan_details
       assert_equal @repository.product.sync_plan.interval, result['product']['sync_plan']['interval']
       assert_equal @repository.product.sync_plan.name, result['product']['sync_plan']['name']
       assert_equal @repository.product.sync_plan.sync_date, result['product']['sync_plan']['sync_date']
+      refute_nil @repository.product.sync_plan.next_sync
       assert_equal @repository.product.sync_plan.next_sync, result['product']['sync_plan']['next_sync']
```

Comment:
note that .next_sync was returrning nil, which was throwing a deprecation warning:

```
DEPRECATED: Use assert_nil if expecting nil from ./katello/test/controllers/api/v2/repositories_controller_test.rb:471. This will fail in Minitest 6.
```

but i also suspect was causing failures on foreman prs (although i don't have much evidence to support that)